test_files/index_import/user.tsickle.ts

```diff
@@ -8,7 +8,7 @@ import {a} from './has_index/index';
 const tsickle_forward_declare_1 = goog.forwardDeclare("test_files.index_import.has_index.index");
 export {a} from './has_index/index';
 const tsickle_forward_declare_2 = goog.forwardDeclare("test_files.index_import.has_index.index");
-export {a} from './has_index/index';
+export {} from './has_index/index';
```

Comment:
@evmar I think this change is harmless and/or intended, we just avoid assigning the same export twice. I think this probably was an oversight of the previous code, which was confused about the `export {a}` also having an `import {a}` here.